### PR TITLE
Added support for sequences and an example to the README

### PR DESCRIPTION
**Pull request by ejlangev**
This feature exists in the Rails version, while not strictly required it is a nice to have rather than managing random variables which you might forget to increment properly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +71,23 @@ factory.build('post', function(err, post) {
 });
 ```
 
+## Defining Sequences
+```javascript
+factory.define('post', Post, {
+  // Returns the next number in the sequence starting with 1.
+  // Automatically increments for the instance.
```
Comment (aexmachina) [inline README.md]: Second line of comment is unnecessary IMHO.

**Comment by aexmachina** [comment]
Nice one :+1:

**Comment by aexmachina** [comment]
Published v2.1.0

**Comment by ejlangev** [comment]
Wow great! Thanks for the quick response.  I'm thinking of adding support for traits also which would be nice to have.

**Comment by aexmachina** [comment]
Yeah, somebody else submitted an issue for that, see #17.

**Pull request merged**